src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -540,6 +540,7 @@ public Game getByUid(int uid) {
               gameLaunchMessage.getArgs().add("/team " + gameLaunchMessage.getTeam());
               gameLaunchMessage.getArgs().add("/players " + gameLaunchMessage.getExpectedPlayers());
               gameLaunchMessage.getArgs().add("/startspot " + gameLaunchMessage.getMapPosition());
+              gameLaunchMessage.getArgs().add("/gameoptions " + gameLaunchMessage.getGameOptions());
```

Comment:
How does java handle `String + Map`? I have a feeling there needs to be some conversion happening here to construct the string correctly.

Reply:
Probably need to use a Collectors.joining